client/src/hooks/admin/adminHooks.ts

```diff
@@ -0,0 +1,54 @@
+import { useState } from 'react'
+import toast from 'react-hot-toast'
+import { useRouter } from 'next/router'
+
+import axios from '~/shared/lib/axios'
+import { setData } from '~/utils/setData'
+import { catchError } from '~/utils/handleAxiosError'
+
+const adminHooks = () => {
+  const router = useRouter()
+  const [fetchStatus, setFetchStatus] = useState<any>({
+    isError: false,
+    isSuccess: false,
+    isLoading: false,
+    error: {
+      status: 0,
+      content: null
+    }
+  })
+  const { isError, isSuccess, isLoading, error } = fetchStatus
+  const csrf = () => axios.get('/sanctum/csrf-cookie')
+
+  const changeAdminPassword = async (data: any) => {
+    try {
+      await csrf()
```

Comment:
Remove `await csrf()`. We only call this function initially during login or register.